src/jwplayer/players/shared/JwPlayerWrapper.tsx

```diff
@@ -36,6 +36,7 @@ const JwPlayerWrapper: React.FC<JwPlayerWrapperProps> = ({
 	stopAutoAdvanceOnExitViewport,
 	shouldLoadSponsoredContentList = true,
 	jwPlayerContainerEmbedId = 'featured-video__player',
+	vastXml = '',
```

Comment:
IMO we don't need a default empty string value. Can we just have it undefined as default?